CMakeLists.txt

```diff
@@ -206,11 +206,14 @@ else()
 endif()
 
 if (NOT MSVC)
-    add_compile_options(-Wall
-                        -Wno-unused-function
-                        -Wcast-qual
-                        -Woverloaded-virtual
-                        -Wignored-qualifiers)
+  add_compile_options(-Wall
+                      -Wno-unused-function
+                      -Wcast-qual
+                      -Woverloaded-virtual
+                      -Wignored-qualifiers)
+  if (CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 5.1)
+    add_compile_options(-Wsuggest-override)
```

Comment:
Does this handle Clang as well?

Reply:
@steven-johnson Clang has [`-Winconsistent-missing-override`](https://clang.llvm.org/docs/DiagnosticsReference.html#winconsistent-missing-override) enabled by default, and `-Wsuggest-override` is G++-specific.